executor/src/witgen/machines/mod.rs

```diff
@@ -350,7 +348,47 @@ impl<T: Display> Display for Connection<'_, T> {
     }
 }
 
-impl<T> Connection<'_, T> {
+impl<'a, T: FieldElement> Connection<'a, T> {
+    /// Creates a connection if the identity is a bus send.
+    pub fn try_new(
+        identity: &'a Identity<T>,
+        bus_receives: &'a BTreeMap<T, BusReceive<T>>,
+    ) -> Option<Self> {
+        match identity {
+            Identity::BusSend(bus_interaction) => {
+                let send = bus_interaction;
+                let receive = send
+                    .try_match_static(bus_receives)
+                    .expect("No matching receive!");
+                let multiplicity_column = if receive.is_unconstrained() {
```

Comment:
This multiplicity column is only referenced rarely and I think can be removed in a follow-up PR. For now, I try to preserve the old behavior to minimize code changes.